settings.gradle

```diff
@@ -39,3 +19,5 @@ include ':shark-hprof-test'
 include ':shark-log'
 include ':shark-test'
 include ':leakcanary-deobfuscation-gradle-plugin'
+
+enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
What does this do?

Reply:
See [Gradle 7.0-rc1 Release Notes](https://docs.gradle.org/7.0-rc-1/release-notes.html).

Reply:
Version catalog migration had been finished in many square projects, like [KotlinPoet](https://github.com/square/kotlinpoet/blob/master/gradle/libs.versions.toml), [Retrofit](https://github.com/square/retrofit/blob/master/gradle/libs.versions.toml) did.